# Request
uint32  bytesToRead
bool    publish
---
# Response
int32   bytesReceived
uint8[] data
